### ISSUE TITLE
sequence number doesn't match tiff numbers

### ISSUE THREAD

**Issue by EliotGann** [bluesky/suitcase-tiff#30]
A user just noticed that the seq_num in the csv suitcase is off by one from the tiff numbers in the filenames created from suitcase-tiff.  seq_num starts at 1, tiff suitcases start at 0.  seems like they should be consistent.

**Comment by danielballan** [comment]
Yeah....an Original Sin of bluesky is that we caved to pressure to start the sequence number at 1 "because that's how SPEC does it" even though everything in Python / SciPy land starts at 0. Without taking a position on which is "right", I do think we should have been consistent with Python here.

Can folks weigh in with an opinion on whether the 1-indexing should propagate into the file names?

**Comment by st3107** [comment]
Just curious, if this issue has been settled down. I am writing suitcases to export 1 dimesion numpy array in data to csv files. I would like the file name to be consistent with the other suitcase packages. However, I found that the indexing of suitcase-tiff and suitcase-csv are different. I wonder which indexing method (1-indexing or 0-indexing) will be chosen as the default.
